frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss

```diff
@@ -78,6 +86,7 @@
         &.LemonButton--small {
             --lemon-button-padding-horizontal: 0.5rem;
             --lemon-button-height: 2.0625rem;
+            --lemon-button-side-action-icon-width: 1.25rem;
```

Comment:
Is this not already the default from https://github.com/PostHog/posthog/pull/19117/files#diff-c694fbbe6b6ff7273c964743f37751e5416009c150d0a9ddad1b4c031ef006d3R14

Reply:
Yup, I started this branch before your PR went in. Removed my change, this probably caused a lot of the snapshot changes